metadata_service/proxy/gremlin_proxy.py

```diff
@@ -241,6 +243,10 @@ def get_table(self, *, table_uri: str) -> Table:
                 id=app_node['application_id'],
             )
 
+        last_updated_timestamp = None
+        if result['table_last_updated']:
+            last_updated_timestamp = int(result['table_last_updated'])
```

Comment:
could we validate that this is a valid timestamp? not worth it? does something downstream of this handle that?